apps/api/src/app/layouts/e2e/helpers/index.ts

```diff
@@ -0,0 +1,46 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+
+import { LayoutDto } from '../../dtos';
+import { LayoutName, TemplateVariableTypeEnum } from '../../types';
+
+const BASE_PATH = '/v1/layouts';
+
+export const createLayout = async (session: UserSession, name: LayoutName, isDefault: boolean): Promise<LayoutDto> => {
```

Comment:
Created this helper file as the layout creation functionality is being reused in all the tests.